examples/simulation/Tutorial1/src/FairSimConfig.cxx

```diff
@@ -35,53 +45,58 @@ FairSimConfig::FairSimConfig()
         ("random-seed",    po::value<int>(),            "Seed for the random number generator");
     // clang-format on
-}
-
-FairSimConfig::~FairSimConfig() {}
-
-int FairSimConfig::ParseCommandLine(int argc, char* argv[])
-{
     try {
-        po::store(po::parse_command_line(argc, argv, fDescription), fMap);
+        po::store(po::parse_command_line(argc, argv, description), map);
 
-        po::notify(fMap);
+        po::notify(map);
     } catch (po::error& e) {
         LOG(error) << e.what();
         fHelp = true;
         return 0;
     }
 
-    if (fMap.count("help")) {
+    if (map.count("help")) {
         fHelp = true;
         return 0;
     }
-    if (fMap.count("nevents")) {
-        fnEvents = fMap["nevents"].as<int>();
+    if (map.count("nevents")) {
+        fnEvents = map["nevents"].as<int>();
     }
-    if (fMap.count("engine")) {
-        fEngine = fMap["engine"].as<vector<string>>().at(0);
+    if (map.count("engine")) {
+        fEngine = map["engine"].as<vector<string>>().at(0);
         if (!GetEngine().EqualTo("TGeant3") && !GetEngine().EqualTo("TGeant4")) {
             LOG(error) << "Option engine can be either TGeant3 or TGeant4";
             return 1;
         }
     }
-    if (fMap.count("output-file")) {
-        fOutputFile = fMap["output-file"].as<vector<string>>().at(0);
+    if (map.count("output-file")) {
+        fOutputFile = map["output-file"].as<vector<string>>().at(0);
     }
-    if (fMap.count("parameter-file")) {
-        fParameterFile = fMap["parameter-file"].as<vector<string>>().at(0);
+    if (map.count("parameter-file")) {
+        fParameterFile = map["parameter-file"].as<vector<string>>().at(0);
     }
-    if (fMap.count("random-seed")) {
-        fRandomSeed = fMap["random-seed"].as<int>();
+    if (map.count("random-seed")) {
+        fRandomSeed = map["random-seed"].as<int>();
     }
-    if (fMap.count("multi-threaded")) {
+    if (map.count("multi-threaded")) {
         LOG(info) << "YUPYUPYUP";
         fMultiThreaded = true;
     }
     return 0;
 }
 
-void FairSimConfig::PrintHelpMessage()
-{
-    std::cout << fDescription << std::endl;
-}
+void FairSimConfig::PrintHelpMessage() { 
+  po::options_description description;
+  // clang-format off
+  description.add_options()
+      ("help",                                        "Print this message")
+      ("nevents",        po::value<int>(),            "Number of events to simulate")
+      ("engine",         po::value<vector<string>>(), "Monte Carlo engine")
+      ("multi-threaded",                              "Geant4 multi threaded")
+      ("output-file",    po::value<vector<string>>(), "Output file")
+      ("parameter-file", po::value<vector<string>>(), "Parameter file")
+      ("random-seed",    po::value<int>(),            "Seed for the random number generator");
+  // clang-format on
+  std::cout << description << std::endl; }
+
+ClassImp(FairSimConfig);
```

Comment:
Where is this ClassImp coming from?!

Reply:
Stupid copy and paste. I did the changes in v18.8_patches and copied them over. Fixed.